platforms/core-configuration/configuration-cache/src/test/kotlin/org/gradle/internal/cc/impl/serialization/codecs/AbstractUserTypeCodecTest.kt

```diff
@@ -117,7 +117,6 @@ abstract class AbstractUserTypeCodecTest {
     private
     fun readFrom(inputStream: ByteArrayInputStream, codec: Codec<Any?>) =
         readContextFor(inputStream, codec).run {
-            initClassLoader(javaClass.classLoader)
```

Comment:
❔  Why is this no longer necessary?

Reply:
Because it was never set to a different `ClassLoader` and so the default value could be moved to `DefaultClassDecoder`.